registrations/management/commands/pseudonymize_past_signups.py

```diff
@@ -0,0 +1,43 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils.timezone import localtime
+
+from registrations.models import SignUp, SignUpGroup
+
+
+class Command(BaseCommand):
+    help = (
+        "Pseudonymize signups and signup groups for past events. "
+        "The time of pseudonymization can be specified in days in the "
+        "PSEUDONYMIZATION_THRESHOLD_DAYS environment variable."
+    )
+
+    def handle(self, *args, **options):
+        threshold_time = localtime() - timedelta(
+            days=settings.PSEUDONYMIZATION_THRESHOLD_DAYS
+        )
+        signup_groups = SignUpGroup.objects.prefetch_related("signups").filter(
```

Comment:
Maybe consider using [select_for_update](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#select-for-update) for both of the querysets and running the pseydonymization loops inside transactions. This will prevent scenarios where another process is updating signup data concurrently and that updated value ends up as the value instead of the pseudonymized value.